### DISCUSSION TITLE
Links to crosswords created with Exet

### DISCUSSION THREAD

**Discussion by viresh-ratnakar** [viresh-ratnakar/exet#19] [Show and tell]
Feel free to post your links in this discussion.

My own crosswords (as the setter Gussalufz) are now all constructed using Exet, and can be accessed at [gussalufz.com](https://gussalufz.com).

**Comment by jdacre** [comment]
Hi Viresh,

I'm in love with Exet, and have been really getting in to setting crosswords! I'm not very confident of the quality of the clues as I've not gotten any serious feedback but I'd love to share some of my more recent attempts. I can save them in different formats but not sure how to obtain a general-access web link to them, any advice?

**Reply by viresh-ratnakar** [reply]
Hey, thanks—I'm glad you're enjoying Exet!

You can save as .puz and publish on [mycrosswords](https://mycrossword.co.uk/) perhaps? (I haven't tried this though).

You can also create a blog on blogger for free and embed Exolve crosswords in it. Exet's "Save" menu lets you copy the widget code that you can paste into the blog entry's HTML.

I myself have created a free web site on github pages and I serve my puzzle files (Exolve) from it, but this option does involve a little bit of set-up.

Viresh

**Reply by jdacre** [reply]
Thanks Viresh, I couldn't get the features on Blogger working but have put up a quick cryptic to mycrosswords: https://www.mycrossword.co.uk/quick/31

I shall persevere with Blogger as I like the idea, and hopefully will keep sharing them in future! I've not had much feedback so a little hesitant as I have no idea on the quality of my clues, but happy to spread the word of Exet :)

Cheers,

Josh
